### DISCUSSION TITLE
[1.8.4] 2023-08-12

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#744] [General]
# Bluetooth
*Use Switch default report (0x3F) to mitigate multiplayer issues w/ digital pad only Switch ctrls
*Enable Sniff mode for Switch controller at all time
*Enable Sniff when more than one controller connected
*Added Rumble Support for generic devices (Thanks @JPZV)
*Fix SNES Switch controller mapping regression
*Add PS5 DualSense trigger end button effects (Thanks @@coolbho3k)

# Wired
*GameCube: Fix Metroid Prime trigger issue with BT controller w/ digital only trigger
*Fix inverted logic for Saturn 3D ctrl triggers
*Fix dreamcast mode crash with ESP32_PICO_V3 (but limited to 1P only)
*Fix Parallel 1P 3v3 (OD) image

# Global
*Fix parsing of HID descriptor report omitting the ID
*Fix 32 bits HID usage mask
*Fix extra HID buttons assignment
*Make joysticks & triggers maximum change dynamically if higher than default

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.8.4'>[1.8.4] 2023-08-12</a>.</em>

**Comment by coolbho3k** [comment]
Woot! Awesome work!

**Comment by DragonKumera** [comment]
This version is completely broken. Controllers constantly disconnecting after not even a minute. (exclene gamecube controller) 4 of them. Had to flash back to 1.8.3 for now as I tried all sorts of fixes or troubleshooting and nothing worked. Bright side rumble worked but I truly hope this gets fixed soon.
